src/bindings/lua/jansson-lua.c

```diff
@@ -102,15 +102,16 @@ static int json_object_to_lua_table (lua_State *L, json_t *o)
     json_t *value;
     lua_newtable (L);
 
-    json_object_foreach(o, key, value) {
+    json_object_foreach (o, key, value)
+    {
```

Comment:
clang-format seems to mistake the `json_object_foreach` macro as a function, I wonder how we fix that. (Hopefully the answer isn't add a `// clang-format:` command in the code. Nothing would be worse than adding comments about formatting to your code :stuck_out_tongue_winking_eye: )

Reply:
Fixed with `ForEachMacros: ['json_object_foreach', 'json_array_foreach']`